### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ci-iam-user-tf-module.git ci-iam-user-tf-module
    cd ci-iam-user-tf-module
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md examples/basic_usage/README.md examples/basic_usage/main.tf examples/basic_usage/outputs.tf examples/basic_usage/variables.tf main.tf outputs.tf variables.tf 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — CHANGES_REQUESTED** [review]
There are some more tags to be removed - see https://github.com/cisagov/ci-iam-user-tf-module/search?q=var.tags

**Review by dav3r — APPROVED** [review]
🌶️

File: README.md [hunk 1]
```diff
@@ -33,42 +28,59 @@ module "example" {
...
-| role_max_session_duration | The maximum session duration (in seconds) when assuming the IAM role that allows this IAM user to do whatever it needs to do. | `number` | `3600` | no |
-| role_name | The name to assign the IAM roles that allows allows this IAM user to do whatever it needs to do in the staging and production environments (e.g. TestCIIAMUserTFModule).  Note that a hyphen followed by Staging or Production will appended to this name, so that the staging and production roles are differentiated. | `string` | n/a | yes |
-| staging_role_tags | Extra tags to apply only to the staging role resource | `map(string)` | `{}` | no |
-| tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
-| user_name | The name to associate with the AWS IAM user (e.g. test-ci-iam-user-tf-module). | `string` | n/a | yes |
+| production\_role\_tags | Extra tags to apply only to the production role resource | `map(string)` | `{"GitHub_Secret_Name": "TEST_ROLE_TO_ASSUME", "GitHub_Secret_Terraform_Lookup": "arn"}` | no |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| production\_role\_tags | Extra tags to apply only to the production role resource. | `map(string)` | `{"GitHub_Secret_Name": "TEST_ROLE_TO_ASSUME", "GitHub_Secret_Terraform_Lookup": "arn"}` | no |
```

File: README.md [hunk 2]
```diff
@@ -33,42 +28,59 @@ module "example" {
...
+| role\_description | The description to associate with the IAM roles that allow this IAM user to do whatever it needs to do in the production and staging environments (e.g. Test the cisagov/ci-iam-user-tf-module repository). | `string` | n/a | yes |
+| role\_max\_session\_duration | The maximum session duration (in seconds) when assuming the IAM role that allows this IAM user to do whatever it needs to do. | `number` | `3600` | no |
+| role\_name | The name to assign the IAM roles that allows allows this IAM user to do whatever it needs to do in the staging and production environments (e.g. TestCIIAMUserTFModule).  Note that a hyphen followed by Staging or Production will appended to this name, so that the staging and production roles are differentiated. | `string` | n/a | yes |
+| staging\_role\_tags | Extra tags to apply only to the staging role resource | `map(string)` | `{}` | no |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| staging\_role\_tags | Extra tags to apply only to the staging role resource. | `map(string)` | `{}` | no |
```

**Review by mcdonnnj — APPROVED** [review]
Just two variable descriptions to add periods to and then it LGTM 👍

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: See commit bbbbb73.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit bbbbb73.

**Pull request merged**